doc/SETUP.org

```diff
@@ -123,6 +126,7 @@ If you're not planning to use private configuration (some modules don't need it)
 
 The configuration usually contains paths to the data on your disk, and some modules have extra settings.
 The config is simply a *python package* (named =my.config=), expected to be in =~/.config/my=.
+If you'd like to change the location of the =my.config= directory, you can set the =MY_CONFIG= environment variable. e.g. in your .bashrc add: ~export MY_CONFIG="$HOME/.my/"~
```

Comment:
github rendering seems messed up here... IIRC it doesn't like quotes inside ~ ~ :(

Reply:
ah

suppose typically `MY_CONFIG=$HOME/.my` would work anyways, as long as your username doesnt have spaces in it...

Reply:
Yeah... seems that you're asking for trouble if you do that xD 
Seems that it's prohibited in POSIX thankfully https://unix.stackexchange.com/a/401782/180307